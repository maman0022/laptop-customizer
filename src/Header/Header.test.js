import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Header from './Header'

describe('Header Component',()=>{
  it('the header with a h1 element',()=>{
    const wrapper=shallow(<Header/>);
    expect(toJson(wrapper)).toMatchSnapshot();
  })
})