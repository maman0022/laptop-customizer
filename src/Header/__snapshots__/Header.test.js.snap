// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component the header with a h1 element 1`] = `
<header>
  <h1>
    ELF Computing | Laptops
  </h1>
</header>
`;
